contracts/truefi2/mocks/PowTest.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Let's remove this wrapper contract and test directly against the library. The pow unit test should still be legible even if it has some `toUint()` and `fromUint()` conversion calls.

Reply:
We can't deploy the library and therefore can't unit test it. I'll keep the contract as simple as possible

Reply:
Oops, I missed that it was a library. Makes sense!